automatic/renamemaster/tools/chocolateyInstall.ps1

```diff
@@ -13,8 +12,9 @@ $packageArgs = @{
 
 Install-ChocolateyZipPackage @packageArgs
 
-Install-ChocolateyPinnedItem "$toolsDir\RenameMaster.exe"
-
-Install-ChocolateyExplorerMenuItem 'openRenameMaster' 'Rename Master...' "$toolsDir\RenameMaster.exe"
```

Comment:
Why remove the ability to run the program from the right-click menu in explorer?

This seems to be something that would be expected for this program.

Reply:
I have added this myself in the past. And now I remove it by previously ignored old popular request. 

It is non-default custom behavior that isn't done by the app itself. I have received complaints over the past few years. It's not even done properly because the context menu only appears on _files_ (and annoyingly visible on all files) while it only works on _directories_.

The installation functionality from chocolatey should not deviate from actually running the installer yourself. So I removed this.

Reply:
Fair enough, thanks for the clarification